subprojects/core/src/main/java/org/gradle/execution/plan/ToPlannedNodeConverter.java

```diff
@@ -28,6 +29,7 @@
  * Each implementation of this interface is responsible for nodes of {@link #getSupportedNodeType()}.
  * The converter can obtain the node identity for each node of the supported type via {@link #getNodeIdentity(Node)}.
  */
+@NonNullApi
```

Comment:
I'd prefer doing the annotation on the package and going through the classes in there. Is that a problem?

Reply:
Let's create an issue to follow up here.

Reply:
https://github.com/gradle/gradle/issues/24338